lib/random-junk-food.js

```diff
@@ -1,10 +1,10 @@
 function RandomJunkFood(x){
   var junkFoods = [
-    {x: x, "type": "burger", "color": "saddlebrown"},
-    {x: x, "type": "hotdog", "color": "tan"},
-    {x: x, "type": "cake", "color": "black"},
-    {x: x, "type": "friedChicken", "color": "dimgray"},
-    {x: x, "type": "pizza", "color": "salmon"}
+    {x: x, "type": "burger", "color": "saddlebrown", "score": "5"},
```

Comment:
I like the idea of adding a score to the junk foods also. Let's look up and have another relative scale for different negative amounts for each junk food.